apps/client/src/features/viewers/studio/StudioClock.jsx

```diff
@@ -46,6 +43,10 @@ export default function StudioClock(props) {
   const secondsIndicators = [...Array(60).keys()];
   const MAX_TITLES = 12;
 
+  const [searchParams] = useSearchParams();
+  const showSeconds = searchParams.get('seconds');
+  formatOptions.showSeconds = Boolean(showSeconds);
```

Comment:
To consider the 12 hour format, we would need to also format conditionally, like
```
  const [searchParams] = useSearchParams();
  const showSeconds = searchParams.get('seconds');
  formatOptions.showSeconds = Boolean(showSeconds);
  formatOptions.format = `hh:mm${formatOptions.showSeconds ? ':mm' : ''}`;
```